ouroboros-consensus-byron-test/test/Test/ThreadNet/Byron.hs

```diff
@@ -1290,26 +1289,26 @@ mkRekeyUpd
   => Genesis.Config
   -> Genesis.GeneratedSecrets
   -> CoreNodeId
-  -> ProtocolInfo m ByronBlock
+  -> ProtocolInfo ByronBlock
+  -> m [BlockForging m ByronBlock]
   -> EpochNo
   -> Crypto.SignKeyDSIGN Crypto.ByronDSIGN
   -> m (Maybe (TestNodeInitialization m ByronBlock))
-mkRekeyUpd genesisConfig genesisSecrets cid pInfo eno newSK =
-    pInfoBlockForging pInfo <&> \blockForging ->
-      case listToMaybe blockForging of
-        Nothing -> Nothing
+mkRekeyUpd genesisConfig genesisSecrets cid pInfo blockForging eno newSK = do
+  listToMaybe <$> blockForging >>= \case
```

Comment:
Minor: you could leave it as `<&>`, I think.

Reply:
Fixed.